src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4362,12 +4362,6 @@ private static bool IsValidAccessToScopedProperty(IExternalControl lhsControl, I
                    (currentControl.IsComponentControl ||
                    (currentControl.TopParentOrSelf is IExternalControl { IsComponentControl: false })))
                 {
-                    // Behavior property is blocked from outside the component.
-                    if (isBehaviorOnly)
-                    {
-                        return false;
```

Comment:
I this something we should (or even can) write a test for?

Reply:
In this codebase, anyway

Reply:
Not at this point.